ci.sh

```diff
@@ -12,6 +12,14 @@ set -ex
   cargo fmt -- --check
   cargo clippy -- --deny=warnings
 )
+( cd dpe/fuzz
+  rustup toolchain install nightly-2023-04-15
+  cargo +nightly-2023-04-15 install cargo-fuzz cargo-afl
+  cargo fmt --check
+  cargo clippy
```

Comment:
Looks like clippy needs one of the features enabled to succeed

Reply:
Oh, right. May as well check both of them, this crate includes different code depending on features.